README.md

```diff
@@ -88,14 +96,27 @@ For general use, use the `main` branch along with [version 2.15.4 of the CodeQL
 1. Analyze your CodeQL database:
     
     CodeQL's analysis output is provided in the form of a SARIF log file. For a human readable format, drop the SARIF file into [SARIF Viewer Website](https://microsoft.github.io/sarif-web-component/). (If there are violations, they will show up. If not, the page will not update.)
+
+    CodeQL query suites are provided in the suites directory and contain the sets of all recommended and mustfix queries. The desired query suite file should be downloaded/copied locally.
```

Comment:
Is it correct that we are having people download the suites, or are we moving to having people download the bundle and then just need to run one CLI command per your convo with GitHub earlier?

Reply:
They will still need to download the suites as I found out there are still issues with using the bundle